spec/support/vcr_cassettes/sm/messages/10616687/index.yml

```diff
@@ -41,7 +41,7 @@ http_interactions:
         13 Apr 2016 21:17:57 GMT","senderId":384939,"senderName":"MVIONE, TEST","recipientId":35382,"recipientName":"Manivasagam,
         Rajesh","readReceipt":"READ"},{"id":573052,"category":"OTHER","subject":"Release
         16.2- SM last login ","body":null,"attachment":false,"attachments":null,"sentDate":"Mon,
-        11 Apr 2016 15:49:03 GMT","senderId":384939,"senderName":"MVIONE, TEST","recipientId":257555,"recipientName":"ISLAM,
+        11 Apr 2016 15:49:03 GMT","senderId":384939,"senderName":"MVITWO, TEST","recipientId":257555,"recipientName":"ISLAM,
```

Comment:
The messages index has only 2 records. Of the filterable fields (sent_date, sender_name, and subject), only date was unique. At the time, I wanted to have some choice when implementing substring match testing, which we've since put off, and made the sender unique too. Since it didn't affect any specs, I left it.